docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -0,0 +1,20 @@
+---
+id: 870-to-880
+title: Update 8.7 to 8.8
+description: "Review which adjustments must be made to migrate from Camunda 8.7.x to Camunda 8.8.0."
+---
+
+The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x for each component.
+
+## Removed deprecated OpenAPI objects
+
+In previous releases we gradually transformed our entity keys from `number (int64)` to `string` types while still
+providing users the opportunity to keep using the deprecated versions. However, with 8.8 release we removed the already
+deprecated API objects containing number keys and with that, the additional content type headers, too.
+
+If you already switched to `string` keys, then this change will not affect you in any way.
+
+If you haven't updated yet, then now it's time.
+You can read the details about this in the [overview about API Key Attributes][camunda8-api-overview].
+
+[camunda8-api-overview]: /apis-tools/camunda-api-rest/camunda-api-rest-overview.md/#api-key-attributes
```

Comment:
🔧 We also need to adjust the 8.8 documentation linked here, right? We still describe the different content types in there and that might be confusing since you cannot use them anymore. We could add a hint there that previously those existed but cannot be used anymore. With 8.9, we might as well remove that hint, too.

Reply:
I'm thinking. Theoretically in 8.8 we don't need the Key attribute section in the overview anymore but in the operational guide it would be nice to provide a link with details.
Do you think it's fine to link the 8.7 overview?

Reply:
Makes sense, yeah, good idea 👍